server/prometheus.js

```diff
@@ -80,6 +105,76 @@ class Prometheus {
             }
         }
 
+        if (uptime) {
+            if (typeof uptime.avgPing !== "undefined") {
```

Comment:
I have no clue what war crimes we did commit here.
(Looking at the surrounding code, this fits in..)

I have no clue why the whole `typeof` verifcation exists in this class.
Seems strange.

We know what the type we pass into this function is..